site_v5.yml

```diff
@@ -43,67 +43,77 @@
         version: "Unknown"
         sha256sum: ae4185b2f0bb1af00abc6a4502fbfbdc6a90aec65c7bcee08e37a1bc20de5ac1 
-      logback_access:
-        baseurl: "{{ aaf_binaries.baseurl }}/jars/logback"
-        version: 1.2.10
-        sha256sum: 8462a7d983a10700c2cc559b7df4254f3a21c2611c32476cb5a6702ce51524c9
+      jcl_over_slf4j:
+        baseurl: "{{ aaf_binaries.baseurl }}/jars/slf4j"
+        version: 2.0.5
+        sha256sum: 95fa90eb540df07b6a2fa70f3613761180d07c9c8f1c299e3f9a5c943313a2d6
       logback_classic:
         baseurl: "{{ aaf_binaries.baseurl }}/jars/logback"
-        version: 1.2.10
-        sha256sum: 3160ae988af82c8bf3024ddbe034a82da98bb186fd09e76c50543c5b9da5cc5e
+        version: 1.3.6
+        sha256sum: c4958cc76d6197ad7ad461fa7be549e3098267d6ff98c831c672d3a4161f5b22
       logback_core:
         baseurl: "{{ aaf_binaries.baseurl }}/jars/logback"
-        version: 1.2.10
-        sha256sum: ba51a3fe56691f9dd7fe742e4a73c3ab4aaaa32202c73409ba56f18687399a08
-      jcl_over_slf4j:
-        baseurl: "{{ aaf_binaries.baseurl }}/jars/slf4j"
-        version: 1.7.33
-        sha256sum: a2c0b70507e6f769ea571608fd9b728329c5c723af37c3b772127c58fb593dfc
-      slf4j_api:
-        baseurl: "{{ aaf_binaries.baseurl }}/jars/slf4j"
-        version: 1.7.33
-        sha256sum: 336811855ee97fcd8413343dc42f4d26362502860e8a9193e4db305e3e8ef80a
+        version: 1.3.6
+        sha256sum: 671500d3e0f998aeeb412bcd9088119ef8a7013a4d708d356bd4b0afb3c02f64
+      spring_jdbc:
+        baseurl: "{{ aaf_binaries.baseurl }}/jars/spring-jdbc"
+        version: 6.0.11
+        sha256sum: 3f617c3999e2fa27b67a07e5e64d81c2f0e10bb39f6fe9fcfedc28e331ec752b
+      plugins:
+        baseurl: "{{ aaf_binaries.baseurl }}/plugins"
+        nashorn:
+          name: "net.shibboleth.idp.plugin.nashorn"
+          version: 2.0.0
+          sha256sum: 6de49acf3a53fbec8698f1e985a0b7b2d16605d1a1a84f8ebf65a6476ba85f2a
+        storage_jdbc:
+          name: "net.shibboleth.plugin.storage.jdbc"
+          version: 2.0.0
+          sha256sum: abac557a582c344c990b5f270c3d84fbe93f8f93d52894057bf50ed037b476bd
     urls:
       jetty:
-        url: "{{ download.jetty.baseurl }}/jetty-distribution-{{ download.jetty.version }}.tar.gz"
+        url: "{{ download.jetty.baseurl }}/jetty-home-{{ download.jetty.version }}.tar.gz"
       shib_idp:
         url: "{{ download.shib_idp.baseurl }}/shibboleth-identity-provider-{{ download.shib_idp.version }}.tar.gz"
       mysql_connector:
         url: "{{ download.mysql_connector.baseurl }}/mysql-connector-java-{{ download.mysql_connector.version }}.tar.gz"
       dta_ssl:
         url: "{{ download.dta_ssl.baseurl }}/jetty94-dta-ssl-{{download.dta_ssl.version}}.jar"
       aaf_shib_ext:
-        url: "{{ download.aaf_shib_ext.baseurl }}/aaf-shib-ext-{{ download.aaf_shib_ext.version }}.jar"
+        url: "{{ download.aaf_shib_ext.baseurl }}/aaf-shibext-{{ download.aaf_shib_ext.version }}.jar"
```

Comment:
Is this a typo?
```suggestion
        url: "{{ download.aaf_shib_ext.baseurl }}/aaf-shi-bext-{{ download.aaf_shib_ext.version }}.jar"
```

Reply:
Not a typo. New version based to the NZ code, so new directory.